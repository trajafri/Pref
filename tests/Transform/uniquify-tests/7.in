(define f (lambda (f) (f 2)))

(define g (lambda (f) 3))

(let ((f (f g)))
  (+ f 3))
