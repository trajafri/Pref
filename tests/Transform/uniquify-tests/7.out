(define f.0 (lambda (f.1) (f.1 2)))

(define g.0 (lambda (f.2) 3))

(let ((f.3 (f.0 g)))
  (+ f.3 3))
